changelog/upgrade-notes.md

```diff
@@ -6,6 +6,11 @@
 
 The experimental `setHue`, `setContrast`, `setBrightness`, `setSaturation`, and the corresponding getter methods have been removed.  These properties only worked with the WebGL renderer.  If are interested in applying color transforms, look for the `postcompose` event in the API docs.  In addition, the `ol.source.Raster` source provides a way to create new raster data based on arbitrary transforms run on any number of input sources.
 
+#### Removal of `ol.ENABLE_NAMED_COLORS`
+
+This option existed to support the use of Google Closure library named colors and was removed as a part of OpenLayers moving away from Google Closure library. Note that this do not affect the use of CSS named colors.
```

Comment:
How about:

Note that this option default to `false` and that it has no effect on the canvas renderer (which is the default). If you have relied on this option to use named colors (created with `ol.color.fromString`) for use with the WebGL renderer you will need to translate them to RGB values.